app/assets/src/components/visualizations/heatmap/heatmap.scss

```diff
@@ -1,5 +1,7 @@
 @import "~styles/themes/colors";
 @import "~styles/themes/typography";
+@value colors: "../../../styles/themes/_colors.scss";
```

Comment:
cant use `~styles/`?

Reply:
Apparently not 😞
![image](https://user-images.githubusercontent.com/53838890/65188868-38e94400-da24-11e9-8242-d52e7ecbc6e5.png)
![image](https://user-images.githubusercontent.com/53838890/65188873-3b4b9e00-da24-11e9-9d03-2231c58dee46.png)